ui/src/app/pages/data/data-set/dataset-page.tsx

```diff
@@ -1104,6 +1107,15 @@ export const DataSetComponent = fp.flow(withUserProfile(), withCurrentWorkspace(
       </div>;
     }
 
+    onClickExport() {
+      this.setState(state => {
+        return {
+          ...state,
```

Comment:
nit: this can be omitted, React does this by default if you omit keys from the state object

Reply:
oh cool. That makes a lot of sense